cmake/SundialsTPLOptions.cmake

```diff
@@ -61,6 +61,11 @@ sundials_option(ENABLE_HIP BOOL "Enable HIP support" OFF)
 # -------------------------------------------------------------
 sundials_option(ENABLE_SYCL BOOL "Enable SYCL support" OFF)
 
+sundials_option(SUNDIALS_SYCL_2020_UNSUPPORTED BOOL
+                "Disable some uses of SYCL 2020 features" OFF
```

Comment:
This CMake option description is perhaps overly vague.  I realize that there are space limitations, but could this be a least a bit more descriptive?